app/uk/gov/hmrc/exports/migrations/changelogs/cache/RenameToAdditionalDocuments.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.migrations.changelogs.cache
+
+import java.util
+
+import scala.collection.JavaConverters._
+
+import com.mongodb.client.MongoDatabase
+import org.bson.Document
+import org.mongodb.scala.model.Filters.{and, exists, not, size, eq => feq}
+import org.mongodb.scala.model.UpdateOneModel
+import org.mongodb.scala.model.Updates.set
+import play.api.Logger
+import uk.gov.hmrc.exports.migrations.changelogs.{MigrationDefinition, MigrationInformation}
+
+class RenameToAdditionalDocuments extends MigrationDefinition {
+
+  private val logger = Logger(this.getClass)
+
+  private val INDEX_ID = "id"
+  private val INDEX_EORI = "eori"
+
+  private val collectionName = "declarations"
+
+  override val migrationInformation: MigrationInformation =
+    MigrationInformation(
+      id = "CEDS-3254 Rename items.documentsProducedData to items.additionalDocuments",
+      order = 9,
```

Comment:
`order` should actually be 3, because the last migration created was about splitting Notifications collection. The one with no. 8 can be removed in my opinion. It is a left-over from the previous implementation and was useful as a quick look-up for implementing new ones. But since we have the 3rd migration now, it serves no purpose anymore.

Reply:
:+1: